docs/docs/recce-cloud/setup-gh-actions.md

```diff
@@ -119,6 +119,16 @@ jobs:
 If executed successfully, the dbt target will be placed in the run artifacts and will be named **target**.
 ![alt text](../../assets/images/recce-cloud/dbt-artifacts.png){: .shadow}
 
+### Upload base artifacts to Recce Cloud
+For Recce Cloud users, you can upload the base artifact to Recce Cloud by `recce cloud upload-artifacts`. 
+You and other developers on your team can then download by `recce cloud download-base-artifacts` without preparing base artifacts every time. 
+
+If you use dbt Cloud to run CI/CD, and
+
+1. if you have a prod job but not schedule, you can have a GitHub Action Workflow to trigger the job and then [download dbt artifacts](https://docs.getdbt.com/dbt-cloud/api-v2#/operations/Retrieve%20Run%20Artifact) and upload to Recce Cloud.
```

Comment:
We don't have this GitHub action yet according to [2024/12/6 discussion](https://www.notion.so/infuseai/understand-DRC-916-for-PLA-248-15379451d35780f1a26afde7982ecc18?pvs=4#15379451d35780b69aeed7608c82d507)

Reply:
The current example workflow of `base-workflow-main-branch` section is for dbt-core usage.
Yes, we can update the workflow to upload artifacts to Recce Cloud.

And adding another section for dbt-cloud usage.

Reply:
ok. Will try

Reply:
sorry, which one we want to change? 
1. the whole workflow? like below: 
   Fore dbt Core users, this workflow will perform the following actions:
   1. Run dbt on the base environment.
   2. Upload the generated DBT artifacts to [github workflow artifacts](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) for later use.
2. only the dbt artifacts part? 
   This workflow will perform the following actions:
   1. Run dbt on the base environment.
   2. For dbt Core users, upload the generated DBT artifacts to [github workflow artifacts](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) for later use.
   3. For dbt Cloud users, we will provide another github workflow artifacts soon.
  
And, what is "github workflow artifacts", is that "github actions workflow" we defined at [2024/12/5 meeting](https://www.notion.so/infuseai/understand-DRC-916-for-PLA-248-15379451d35780f1a26afde7982ecc18?pvs=4#15479451d35780e4bd4ee87a7794a175)?

Reply:
If we will have 2 github workflow artifacts, can we have different names?

Reply:
resolved and update